lib/edmunds/vehicle/specification/style.rb

```diff
@@ -9,13 +9,21 @@ module Vehicle
     module Specification
       module Style
         class Style
-          attr_reader :id, :name, :trim, :body, :year, :make_name, :model_name
+          attr_reader :id, :name, :trim, :body, :year, :make_name, :model_name, :engine, :transmission, :driven_wheels, :colors, :options
 
-          def initialize(attributes)
+          def initialize(attributes, view = nil)
```

Comment:
I'm sorry, what would be the new params style? Like adding that 'view' option inside the attributes hash?